.changeset/selfish-seals-vanish.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/techdocs-common': minor
+'@backstage/plugin-techdocs-backend': minor
+---
+
+OpenStack Swift Client changed with Trendyol's OpenStack Swift SDK
```

Comment:
I wonder if its worth add a link directly to the migration guide here as its required when upgrading to this new version (as far as I understand)?